test/engines/misc/sends/screen.t.cpp

```diff
@@ -0,0 +1,121 @@
+#include <algorithm>
+#include <cmath>
+#include "core/ui/vector_graphics.hpp"
+#include "testing.t.hpp"
+#include "graphics.t.hpp"
+
+TEST_CASE("Test sends screen" * doctest::skip()) {
+  otto::test::show_gui([&](otto::nvg::Canvas& ctx){
+    using namespace otto::core::ui;
+    using namespace otto::core::ui::vg;
+    ctx.group([&] {
```

Comment:
consider removing this now that the code has been moved to the real implementation

Reply:
Deleted this file